tests/coverage/test_processor.py

```
@@ -0,0 +1,436 @@
+import pytest
+import json
+import xml.etree.ElementTree as ET
+from coverage_ai.coverage.processor import (
+    CoverageProcessor,
```

Ruff (0.8.2)

`coverage_ai.coverage.processor.CoverageProcessor` imported but unused

Remove unused import: `coverage_ai.coverage.processor.CoverageProcessor`

(F401)